src/main/thrift/parquet.thrift

```diff
@@ -474,6 +474,16 @@ enum PageType {
   DATA_PAGE_V2 = 3;
 }
 
+/**
+ * Enum to annotate whether lists of min/max elements inside column indexes
```

Comment:
I'd refer to the actual struct.

Reply:
Change the comment.

I thought that Statistics.distinct_count still may be helpful in row groups, for example a reader may decide whether to read the data into dictionaries on the fly. I think @rdblue wanted to double check that no writer currently uses distinct count in pages and then we would deprecate the page statistics in a subsequent change.

I'm also happy to do either of those in the current PR. @rdblue, @julienledem - what do you think.

Reply:
Good point.

Probably best to leave stats-related changes to a separate PR and get this in.